### PR TITLE
Add samples to the documentation

### LINKED ISSUE #537: Adds samples to the documentation
There are few samples about how to use the annotations and also about special cases.

Additional samples should be added.

For instance:

sample

### PR DESCRIPTION
**Pull request by stockiNail**
Fixes #537

### TODO

- [x] box samples
- [x] ellipse samples
- [x] label samples
- [x] line samples
- [x] point samples
- [x] polygon samples
- [x] remove previous types samples
- [x] additonal samples searching from discussions and issues

### CODE REVIEW

File: docs/samples/line/limited.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Limited lines
+
+```js chart-editor
+// <block:setup:4>
+const DATA_COUNT = 12;
+const MIN = 10;
+const MAX = 100;
+
+Utils.srand(8);
+const jointValue = Utils.rand(MIN, MAX);
```
Comment (kurkle) [inline docs/samples/line/limited.md:10]: jointValue is not used

Comment (stockiNail) [reply docs/samples/line/limited.md]: done

**Review by kurkle — APPROVED** [review]

**Comment by kurkle** [comment]
Huge job you did there, cheers!

**Comment by stockiNail** [comment]
> Huge job you did there, cheers!

Thank you! Some of them were already ready in my lib for GWT/J2CL... I have found another use case, #145. I am thinking how to do it in best way. I don't like to have shadows on top and bottom...

**Comment by stockiNail** [comment]
> I have found another use case, #145. I am thinking how to do it in best way. I don't like to have shadows on top and bottom...

Apart the suggested workaround, that sounds good to me, I think it could be implemented as new enhancement, adding `shawdowBlur` and `shadowColor` as possible options.

I don't add any other use case (if anything is missing, tell me!)  to this PR.

**Pull request merged**